app/models/automated_checks/claim_verifiers/alternative_identity_verification.rb

```diff
@@ -0,0 +1,93 @@
+module AutomatedChecks
+  module ClaimVerifiers
+    class AlternativeIdentityVerification
+      TASK_NAME = "alternative_identity_verification".freeze
+
+      def initialize(claim:)
+        @claim = claim
+      end
+
+      def perform
+        return unless eligibility.claimant_identity_verified_at?
+        return if task_exists?
+        return unless all_details_match?
+
+        create_task!
+      end
+
+      private
+
+      attr_reader :claim
+
+      delegate :eligibility, to: :claim
+
+      def create_task!
+        task = claim.tasks.build(
+          name: TASK_NAME,
+          manual: false,
```

Comment:
was technically a manual test. does setting to `true` have any effect or meaning?